docs/hub/spaces-gpus.md

```diff
@@ -118,4 +118,8 @@ The following options are available:
 <div class="flex justify-center">
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-sleep-time-options.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-sleep-time-options-dark.png"/>
-</div>
\ No newline at end of file
+</div>
+
+## Pausing a Space[[pause]]
```

Comment:
I'm not sure this doc fits the best in the GPU page. This is enabled for all repos, right? Maybe https://huggingface.co/docs/hub/spaces-overview or https://huggingface.co/docs/hub/spaces-settings could be a better fit

Reply:
Added a mention to this in spaces-overview too!